lib/process-services/process-list/components/start-process.component.ts

```diff
@@ -95,16 +102,21 @@ export class StartProcessInstanceComponent implements OnChanges {
         }
     }
 
-    compareProcessDef = (processDefId) => {
-        if (this.processDefinitions && this.processDefinitions.length === 1 && processDefId === this.processDefinitions[0].id) {
-            this.onProcessDefChange(processDefId);
-            return true;
+    selectDefaultOption() {
+        if (this.hasSingleProcessDefinitions()) {
+            this.currentProcessDef.id = this.processDefinitions[0].id;
+            this.selectPanelClass = 'hidden';
+            this.onProcessDefChange();
+        } else if (this.processDefinitionId) {
+            this.currentProcessDef.id = this.processDefinitionId;
+            this.onProcessDefChange();
```

Comment:
we don't need to call onProcessDefChange
it should be automatically called when the model change
(ngModelChange)="onProcessDefChange($event)"